frontend/components/Navbar/index.tsx

```diff
@@ -16,17 +17,22 @@ export const Navbar = (): JSX.Element => {
   const SwitchIcon = useColorModeValue(FaMoon, FaSun);
   const nextMode = useColorModeValue("dark", "light");
 
+  const textColor = useColorModeValue("white", "white");
+  const boxBg = useColorModeValue("#343A40", "#343A40");
+  const buttonBg = useColorModeValue("#2D9CDB", "#2D9CDB");
+  const iconColor = useColorModeValue("#ECC94B", "light");
+
   return (
-    <Flex w="100%" m="2" mb="4">
-      <Box m="2">
+    <Flex w="101%" mb="4" bg={boxBg} boxShadow="lg">
```

Comment:
Suggestion: Since this component is used outside of the main landmark, we need to provide a semantic landmark HTML element as the wrapper rather than it being a div, can we convert this to use a nav?
```suggestion
    <Flex w="101%" mb="4" bg={boxBg} boxShadow="lg" as="nav">
```
For reference: https://www.w3.org/TR/wai-aria-practices/examples/landmarks/HTML5.html